Test.Common.props

```diff
@@ -35,7 +34,7 @@
              Link="profiler-lib\integrations.json" />
   </ItemGroup>
 
-  <Target Name="AfterBuildCopyManagedProfiler" AfterTargets="AfterBuild" Condition=" '$(LoadManagedProfilerFromProfilerDirectory)' == 'true'">
+  <Target Name="AfterBuildCopyManagedProfiler" AfterTargets="AfterBuild">
```

Comment:
I'm removing `$(LoadManagedProfilerFromProfilerDirectory)` because it's not intuitive when to use it. For now, while wasteful, we'll always copy all managed binaries to a sample app's `profiler-lib` directory.